apps/cli/src/commands/hash.ts

```diff
@@ -0,0 +1,22 @@
+import { Command } from "@oclif/core";
+import fs from 'fs';
+import path from "path";
+
+
+export default class HashCommand extends Command {
+    
+    public async run() {
+        // read hash of the cartesi machine snapshot, if one exists
+        const hashPath = path.join(".sunodo", "image", "hash");
+
+        if(fs.existsSync(hashPath)) {
+            const hash = fs.readFileSync(hashPath).toString("hex");
+
+            this.log(hash)
```

Comment:
We should also support json output. Take a look at https://oclif.io/docs/json
Do a `if (this.jsonEnabled())`.
It's similar to what it's done at the AddressBook command, but not the same (you don't need to use `ux.table` in this case.

Reply:
done

Reply:
Replace this line log with:
```
process.stdout.write(
        `${chalk.green("?")} Machine hash ${chalk.cyan(templateHash)}\n`,
    );
```